Add: Privacy page

abdullahlko opened the pull request:
## 🔖 PR Title:
<!-- A clear and concise title e.g. "Fixed leaderboard score update issue in PHP backend" -->

---

## 📄 Description:
<!-- Describe in detail what you changed or added -->

- [ ] Bug Fix ✅
- [ ] New Feature ✨
- [ ] Enhancement 🔧
- [ ] Documentation 📝
- [✔️] UI/UX Update 🎨

---

## 🧑‍💻 What changes were made?
<!-- Bullet points of major changes -->
- Designed and developed the complete Privacy Policy page from scratch
- Added top padding to prevent navbar from overlapping content
- Verified layout on Privacy Policy page in both light and dark mode
---

## 🖼️ Screenshot or Screen Recording (max 15 seconds)
https://github.com/user-attachments/assets/57fd4df2-9caf-44d9-ab52-37ffecd75787

---

## 🔗 Issue Reference:
<!-- Mention the issue number this PR fixes -->
Closes #34
---

## 🧩 Contribution Type:
- [✔️ ] Frontend 🎯
- [ ] Backend 🔙
- [ ] UI/UX 🎨
- [ ] Documentation 📘
- [ ] Bug Fix 🐛
- [ ] Feature Implementation 🚀
- [ ] Optimization ♻️
- [ ] Other:

VAIBHAVBABELE commented:
Resolve conflicts first

VAIBHAVBABELE commented:
Make footer same for all pages of websites. And make similar css for privacy page which for home page @abdullahlko

abdullahlko commented:
Changes implemented :

Fixed dark mode styling to properly reflect dark theme

Removed all inline/internal CSS and moved to index.css

Made footer layout and style consistent with homepage
